pkg/config/environment_detection.go

```diff
@@ -31,6 +31,8 @@ const (
 	ECSFargate
 	// EKSFargate environment
 	EKSFargate
+	// OrchestratorExplorer can be enabled
+	OrchestratorExplorer
```

Comment:
In order to be future proof here, it could be interesting to name it `K8sOrchestratorExplorer`, so we can add `ECSOrchestratorExplorer` and others. This applies to the `ad_identifiers` field as well, `_orchestrator` -> `_k8s_orchestrator` wdyt?